lib/msal-browser/src/interaction_client/SilentIframeClient.ts

```diff
@@ -123,9 +128,11 @@ export class SilentIframeClient extends StandardInteractionClient {
             this.apiId
         );
 
+        let authClient: AuthorizationCodeClient | null = null;
```

Comment:
Leaving it undefined should be acceptable
```suggestion
        let authClient: AuthorizationCodeClient;
```

Reply:
Hmmm I did try that initially but I get a `Variable 'authClient' is used before being assigned. ts(2454)` error due to it being used in the catch block below.

Reply:
You might try `| undefined` then - it may seem trivial but philosophically `null` should be reserved for cases where we want to really explicitly say this is not defined. In this case it should never be left in a falsey state outside of errors initializing it so undefined is more appropriate

Reply:
Fair enough. Updated.